fluent-syntax/src/util.js

```diff
@@ -0,0 +1,3 @@
+export function includes(arr, elem) {
+  return arr.indexOf(elem) > -1;
```

Comment:
could we get a compat-utils that are included only in the compat build? :) I think it would be nice to invest a bit of our time to design a build system for fluent which does not penalize evergreens for the sins of their ancestors. Although I agree that in this case the polyfill is small :)